components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.scss

```diff
@@ -0,0 +1,159 @@
+@import "~styles/variables";
+@import "@carbon/grid/scss/grid";
+
+:host {
+  @extend .bx--grid;
+  position: absolute;
+  top: calc(100% + 8px);
+  right: 0;
+  padding: 16px;
+  border-radius: 8px; // not variable
```

Comment:
There are some values that are not yet slated to be app wide variables that maybe should be - need some further discussion on this.  There are a few yet to be named colors as well that I'd like to talk with the team about - we definitely need them but should talk about convention.

Reply:
This is solid - i'll convert 👍